volumina/imageView2D.py

```diff
@@ -244,14 +245,14 @@ def _setdeaccelerateAxAy(self, x, y, a):
         y = abs(y)
         if x > y:
             if y > 0:
-                ax = int(x / y)
+                ax = x // y
                 if ax != 0:
```

Comment:
Just a small note: In this change, the new line is not equivalent to the old line because the return type changes from int to float.  As it turns out, it doesn't matter in this case because the return value is eventually converted to float anyway.  But I had to go hunting for that to verify that it wasn't a problem.

For this exercise, when the original code achieved integer division via an explicit cast, let's just leave it alone.  It's not wrong, even if it might be a little verbose in some cases.